.github/workflows/linux_tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: Functional tests
+name: Linux Tests
```

Comment:
I was seeking to make the names of the various functional test suites consistent, but I found that the "Functional" part was redundant, because the test jobs appear as `Linux Functional Tests / Functional test (xxxx, yyyy)` and this was getting a bit long. But I can certainly put it back if you think it helps.

Reply:
Ah in that case please disregard this comment. As long as the CI job mentions "functional" so they don't get confused with unit tests, there's no need for duplicating